doc/ExtraDocuments/Conventions.md

```diff
@@ -0,0 +1,196 @@
+
+#### BOOZX
+The transformation function is given back as spectral coefficients over the new Boozer angles. So the it is not function of the VMEC angles ($\theta_V,\varphi_V$) anymore, but of ($\theta_B,\varphi_B$). Comparing to the transformation above, this function is then actually not the transformation 
+
+straight fieldline VMEC coordinates $\rightarrow$ Boozer coordinates
+
+but rather
+
+straight fieldline VMEC coordinates $\leftarrow$ Boozer coordinates
+
+The idea is to have an easy way to map the toroidal Boozer angle $\varphi_B$ to the geometrical angle $\varphi = \varphi_V$. Therefore BOOZX gives the transformation function back as the function $p$
+
+$p \equiv -qG(\Psi_B,\theta_B, \varphi_B)$ &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(Note that $\Psi_B=\Psi_V$).
+
+As a result one can perform the mapping straight-forward
+
+$\varphi = \varphi_B + p(\Psi_B,\theta_B,\varphi_B)$.
+
+#### ITP Boozer
+
+One finds in the `.bc` files of ITP boozer converters (in contrast to the `.nc` files of BOOZX) not the fourier-coefficients $p_{mn}$ but $v_{mn}$. This $v$ is just $p$ with a normalisation
+
+$v \equiv \frac{n_{\text{per}}}{2\pi}*p(\Psi_B,\theta_B,\varphi_B)$.
+
+The prefactor has to be considered when one again wants to perform the mapping to the geometrical angle as above.
+
+#### The point of it all
+
+The output of the Boozer converter gives back the form of the flux surfaces by providing the modes of the cylindrical coordinates $R$ ($R_{mn}$) and $Z$ ($Z_{mn}$) of the flux surfaces. One can evaluate the spectral series $R(\theta_B,\varphi_B)$ and $Z(\theta_B,\varphi_B)$ then for multiple angles to get points on the surface. To get the surface in real space on needs to go to cartesian coordinates however, therefore needing the mapping $\varphi_B \rightarrow \varphi$ to calculate
+
+$X = R*\cos{\varphi}$ <br>
+$Y = R*\sin{\varphi}$ <br>
+$Z = Z$
+
+### Change of Spectral modes
+
+#### Inheriently different coordinate systems
+
+e.g.
+- VMEC > BOOZX
+- VMEC > ITP Boozer
+
+One has to recompute the fourierdecomposition as the angles are now different. This requires that one evaluates the spectral series on a grid of points in the angle space i.e. get the values of $f(s,u,v)$. In the direct transformation "VMEC > ITP Boozer" done by e.g. [vmec2boozer](https://gitlab.tugraz.at/plasma/codes/vmec2boozer) or [boozer.py](https://github.com/itpplasma/libneo/blob/main/python/libneo/boozer.py) (see python functions of [libneo](https://github.com/itpplasma/libneo/tree/main)), one incorporates the coordinate transfromation into the fourierintegral itself :
+
+$f^B_{mn}(s) = \frac{1}{(2\pi)^2}\int_0^{2\pi}d\theta_V\int_0^{2\pi}d\varphi_V{J_B^*}^{-1}\exp{(-i((m+nq)G))f(s,\theta_V,\varphi_V)}\exp{(-i(m\theta + n\varphi))}$
+
```

Comment:
Recommendation for the future: if you ever need to edit your formulas, you will be happy that you used spaces when writing them.